Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "miden-node"
+version = "0.1.0"
+edition = "2021"
+authors = ["miden contributors"]
+keywords = ["miden", "node", "program"]
+rust-version = "1.67"
+
+[dependencies]
```

Comment:
Notes on the dependencies:

- tonic: the gRPC framework
- prost: protocol buffers code generation, used by tonic
- clap: used for the CLI
- toml: used for the config file
- figment: used to support merging configs from multiple sources
    - environment variables are useful for container deployments, dockerd/postman/kubernetes
    - standard location files is useful for package manager installs (deb/rpm/etc)
    - configuration file is useful for daemon managers, systemd/upstart/initd/etc
- serde: is used by figment and trokio-tracing
- directories: handles the systems' default configuration directories (Max/Linux/Windows/etc)
- tokio: the asyncio framework used by tonic
- tracing/tracing-subscriber: the logging infrastructure used by tokio, supports structured logging and open telemetry
- anyhow: used as a replacement to `std::error` because it provides better defaults